src/forms/reduxForm/reduxTextInput.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Yeah, you can use `shallow` to do the necessary unit test:

```jsx
import React from 'react';
import { shallow } from 'enzyme';
import ReduxTextInput from './ReduxTextInput';

describe('ReduxTextInput', function() {
	const formAttrs = {
		input: {
			label: 'Super Hero',
			name: 'superhero',
			value: 'Wonder Woman and Robin',
			maxLength: 20,
			required: true,
		},
		meta: { error: 'Did you mean Batman and Robin?' },
	};

	it('renders a required HTML <input> with expected attributes for mock data', () => {
		const component = shallow(<ReduxTextInput {...formAttrs} />);

		expect(component).toMatchSnapshot();
	});
});
```

yields this snapshot:

```js
exports[`ReduxTextInput renders a required HTML <input> with expected attributes for mock data 1`] = `
<TextInput
  error="Did you mean Batman and Robin?"
  label="Super Hero"
  maxLength={20}
  name="superhero"
  required={true}
  value="Wonder Woman and Robin"
/>
`;
```

I think the issue that you noticed about the missing `value` in the snapshot was related to the structure of your `formAttrs` not being quite right